trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/compute_point_source_coordinates.py

```diff
@@ -23,73 +24,74 @@ def compute_point_source_coordinates(AoA,thrust_angle,mls,prop_origin):
         N/A  
 
     Inputs:  
-    AoA                   - angle of attack           [rad]
-    thrust_angle          - thrust angle              [rad]
-    mls                   - microphone locations      [m]
-    prop_origin           - propeller/rotor orgin     [m]
+    conditions.
+          
+    network.
+    
+    mls                   - microphone locations       [m]
 
     Outputs: 
-        position vector   - position vector of points [m]
+        position vector   - position vector of points  [m]
 
     Properties Used:
         N/A       
     """  
     
+    
     # aquire dimension of matrix
-    num_cpt         = len(AoA)
+    num_cpt         = conditions._size
```

Comment:
same point about naming being vague and not to SUAVE's covention

Reply:
Again, I had to hack this for the regressions. Normally you can pull this from state.numerics but in the regressions that data structure doesn't exist.